src/helpers/hotkeysManager.js

```diff
@@ -33,6 +33,8 @@ const NOOP = () => {};
  * @property {string} Ctrl_P Print
  * @property {string} PageUp Go to the previous page
  * @property {string} PageDown Go to the next page
+ * @property {string} Up Go to the previous page if not scrolling (refers to the ArrowUp key)
```

Comment:
As described in another comment, the arrow key can also be used when the page is zoomed in(there's a scrollbar). So we need to change the documentation.

How about `Go to the previous page in the single layout mode`? Also I feel like inside the bracket we can just write `ArrowUp` to make it consistent with `Ctrl+Shift+=`.

Reply:
addressed